version.json

```diff
@@ -1,7 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/AArnott/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "2.8.0",
-  "versionHeightOffset": 0,
+  "version": "2.8.0-rc.{height}",
```

Comment:
every new commit will build a unique build which will then be tested by orchestrated pipeline.